packages/core/strapi/lib/Strapi.js

```diff
@@ -225,7 +225,7 @@ class Strapi {
 
     await this.runLifecyclesFunctions(LIFECYCLES.DESTROY);
 
-    this.eventHub.removeAllListeners();
+    this.eventHub.removeAllSubscribers();
```

Comment:
know that we implement our own logic we can use our naming conventions: register/bootstrap/destroy and add a destroy function to the eventHub